Topic 0 html css basics

davidleonel opened the pull request.

davidleonel committed:
The Basics of HTML--Primer Punto

Create a basic .html file with a header displaying "Hello World".

davidleonel committed:
The Basics of HTML--Segundo Punto

Change the header to "My todo list".

davidleonel committed:
The Basics of HTML--Puntos 3, 4, 5

-Add a list of "todo items" for your daily chores.
-Create another .html file. Create a table for your expenses.
-Create another .html file. Add an image, a video, and a sound.

davidleonel committed:
The Basics of HTML--Puntos 6, 7

-Create a "sign up" form with fields for: first name, last name, email,
birthday, a dropdown to choose your favourite sport, and a text-area to
include a small bio for the user. Add a button at the end to submit the
form, and another one to clear the form. Add relevant validation rules
for all fields (like required fields, valid email).
-Test your HTML files in at least Firefox, Chrome, IE, and Chrome for
Android or iOS Safari.

davidleonel committed:
Doctypes & Metatags--Todos los Puntos

-Learn how to write valid, and semantic markup:
-Add doctype to the previously created HTML documents. See what happens
if you remove it.
-Add metatags to the document.
-Add the meta viewport tag. Check what happens in a mobile browser with
or without it.
-Validate your markup: W3C Validator
-Finally, check out The HTML5 Boilerplate, to see a best-practice HTML
document.

davidleonel committed:
CSS_Basics--Puntos 1, 2, 3, 4

-Create a basic index.html file (example provided).
-Create an empty style.css file and link it to the index.html using the
link tag.
-Learn why is a best-practice to use a reset stylesheet here. Then
include normalize.css before linking your style.css.
-Add a basic page structure using HTML as depicted by the following
picture.

davidleonel committed:
CSS_Selectors and properties--Puntos 1, 2, 3

-Add background to the header, footer, aside and nav.
-Adda global font definition (at html element) with a value of 14px,
using a font-family you like.
-Center the header and footer text.

davidleonel committed:
CSS_Specificity--Puntos 1,2,3,4,5,6,7

Todos los apartados completos.

troyax commented:
Just an overall comment, leave this as it is now but we should do all the exercises using english. For the future topics and exercises use english

troyax commented:
Good code, keep going you are doing well. Just a few comments, if any doubt about those comments let me know. You still need to do the layer exercise, the grid system and the media exercise. But you are doing really well and going on time.

davidleonel committed:
Update on previews work.

Fixed all the issues pointed out by tutor.

davidleonel committed:
CSS_2.5.1 The display property Exercise

-Now modify your CSS to reach something similar to the initial layout
asked.

davidleonel committed:
CSS_2.5.2 Layout systems--Not Finished

-Using your own layout system, implement an HTML page based on the
following mock-up (only desktop).

davidleonel committed:
CSS_2.6 Media queries--Not Finished

-Adapt your previous exercises to work on mobile screens

davidleonel committed:
Added Commentaries

Added Commentaries

davidleonel committed:
CSS_2.5.2 Layout systems and 2.6 Media queries DONE

Fix mistakes and finished both sections.

troyax commented:
OK, will merge this because it is closed
